packages/wallets/src/components/WalletCard/WalletCard.scss

```diff
@@ -1,9 +1,20 @@
 @import '../../components/SkeletonLoader/SkeletonLoader.scss';
 
 .wallets-card {
-    display: flex;
-    flex-direction: column;
-    align-items: center;
+    &__carousel-content {
+        background-color: transparent;
+        margin: 0 -0.8rem;
```

Comment:
Is it possible to not use negative margins here. So it's easier to debug the css

Reply:
In this scenario the negative margin is needed to offset the value between the active and inactive wallet width after the embla scaling function